lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3128,7 +3128,9 @@ TypeSystemSwiftTypeRef::ShouldPrintAsOneLiner(opaque_compiler_type_t type,
                                               ValueObject *valobj) {
   auto impl = [&]() {
     if (type) {
-      if (IsImportedType(type, nullptr))
+      auto canonical = GetCanonicalType(type);
+      if (canonical)
+        if (IsImportedType(canonical.GetOpaqueQualType(), nullptr))
```

Comment:
This means that we can't support summary formatters on wrapped C++ types in Swift. I guess that's okay?